### DISCUSSION TITLE
Recommended Vivado version

### DISCUSSION THREAD

**Discussion by pipipipi2002** [NUS-CG3207/labs#1] [Wrangling Vivado] [answered]
May I know which vivado version should we use for this course?

**Comment by NeilBaner** [comment, marked as answer]
We recommend using Vivado 2023.2 for this course. That said, if you have an older version installed already, it is very likely to work just as well. 

Make sure to install the Standard or WebPack (depending on your version) edition, so you don't have to pay for a license.
